### PR TITLE
Fix syncing issues between service worker and offline page

### LINKED ISSUE #10667: Offline page and ServiceWorker out of sync
* SW caches /app-1.js
* /offline-page requests /app-2.js

This happens because, since the last deploy, /offline-page has been revalidated but /service-worker.js has not.

We need to find a way to lockstep these two files, to guarantee users will be caching the same version of assets requested by /offline-page.

Here is our current cache setup:

```
/service-worker.js
< Cache-Control: max-age=3600, stale-while-revalidate=360, stale-if-error=864000, private

/offline-page
< Cache-Control: max-age=60, stale-while-revalidate=6, stale-if-error=864000, private
```

### PR DESCRIPTION
**Pull request by OliverJAsh**
Fixes #10667

To do:
- [ ] Update nginx config to route `/offline-page.json` instead of `/offline-page`: https://github.com/guardian/platform/pull/844
- [ ] Update fastly SSL config to force HTTPS for `/offline-page.json` instead of `/offline-page`: https://github.com/guardian/fastly-edge-cache/pull/493

### CODE REVIEW

File: applications/app/templates/serviceWorker.scala.js [hunk 1]
```diff
@@ -19,18 +19,44 @@
...
-                '/offline-page',
-                '@Static("stylesheets/head.content.css")',
-                '@Static("stylesheets/content.css")',
-                '@Static("stylesheets/print.css")',
-                // Crossword pages use jspm
-                '@StaticJspm("javascripts/core.js")',
-                '@StaticJspm("javascripts/bootstraps/app.js")',
-                '@StaticJspm("javascripts/es6/bootstraps/crosswords.js")'
-            ]);
+        // Fetch page and all assets. Iff all responses are OK then cache all assets and page.
```
Comment (gtrufitt) [inline applications/app/templates/serviceWorker.scala.js]: minor sp - iff

**Comment by gtrufitt** [comment]
:+1:

File: applications/app/templates/serviceWorker.scala.js (hunk 1 above)
Comment (OliverJAsh) [inline applications/app/templates/serviceWorker.scala.js]: it means if and only if: https://en.wikipedia.org/wiki/If_and_only_if

Comment (gtrufitt) [inline applications/app/templates/serviceWorker.scala.js]: oooooh right.

**Pull request merged**